index.html

```diff
@@ -56,17 +60,66 @@ <h2>Multi-Post Stories</h2>
                     </ul>
                     <button class="see" type="button">See Project </button>
                 </section>
-                <div class="card"><img src="assets/Img-2.png" class="img-placeholder" alt="">
-                <h2>Professional Art <br>Printing Data</h2>
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                
+                <div class="card">
+                    <h2>Professional Art <br>Printing Data</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul>
+                        <li>CSS</li>
+                        <li>HTML</li>
+                        <li>Ruby</li>
+                    </ul>
+                
+                    <button class="project" type="button">See Project </button>
+                </div>
+                
+            </div> 
+        </div>
+            
+    
+    
+
+    </div>
+    <div class="container-3">
+        <div class="About">
+            <h1>About Me</h1>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <BUtton class="resume">Get My Resume</BUtton>
+            <img src="assets/Frame.png" alt="">
+            
+            
+            <div class="list-1">
+                <h3 class="lan"> Languages</h3>
                 <ul>
+                    <li>JavaScript</li>
+                    <li>Ruby</li>
                     <li>CSS</li>
                     <li>HTML</li>
-                    <li>Ruby</li>
                 </ul>
-                <button class="project" type="button">See Project </button>
-                </div>
-            </div> 
+            </div>
+            <div class="list-2">
+                <h3 class="fram"> Framworks</h3>
+                <ul>
+                    <li>Bootstrap</li>
+                    <li>Ruby on Rails</li>
+                    <li>RSpec</li>
+                    <li>CapyBara</li>
+                    <li>Selenium</li>
+                </ul>
+            </div>
+            <div class="list-3">
+                <h3 class="skills"> Skills</h3>
+                <ul>
+                    <li>Codekit</li>
+                    <li>Github</li>
+                    <li>Codepen</li>
+                    <li>Gitlab</li>
+                    <li>Terminal</li>
+                </ul>
+            </div>
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Template 2-Main-Mobile](https://user-images.githubusercontent.com/69638013/177016064-c237acdc-0abb-422c-91be-8bf03f83c85a.png) | ![Screenshot_308](https://user-images.githubusercontent.com/69638013/177016032-21ed39e3-ed18-45aa-b67f-33d4facadda6.png)

- [ ] Kindly align the background image to the right, please make your design look like the Figma
- [ ] Please align languages, frameworks and skills to the center